openeo/testing/results.py

```diff
@@ -88,6 +91,129 @@ def _as_xarray_dataarray(data: Union[str, Path, xarray.DataArray]) -> xarray.Dat
     return data
 
 
+def ascii_art(diff_data: DataArray) -> str:
+    scale: int = max(1, int(diff_data.sizes["x"] / 100))
```

Comment:
as noted in https://github.com/Open-EO/openeo-python-client/issues/761#issuecomment-2858602486  I think a size of 100 is too much

I would limit it to e.g. 40x40 or something like that

and while at it: just make this max size/width  a function argument (with a reasonable default)